resources/web/wwi/nodes/utils/WbRay.js

```diff
@@ -0,0 +1,48 @@
+import WbVector3 from './WbVector3.js';
+
+export default class WbRay {
+  constructor(origin, direction) {
+    this.origin = origin;
+    this.direction = direction;
+  }
+
+  intersects(minBound, maxBound, tMin, tMax) {
+    const bounds = [];
+    bounds[0] = minBound;
+    bounds[1] = maxBound;
+    const invDirection = new WbVector3(1.0 / this.direction.x, 1.0 / this.direction.y, 1.0 / this.direction.z);
```

Comment:
do we need to worry about divisions by zero?

Reply:
I don't think so (even so it seems strange).
I checked in Webots and the behavior is the same as in WebotsJS: if we dived by 0 the result is `Infinity`